Skip selector calculation of non-violation results

tsmd opened the pull request:
It takes a very long time to inspect a huge page with a large number of nodes.

According to the official documentation, one of the reasons it takes so long to process is the selector calculation. It takes time because the calculations are performed not only for the items that failed the test, but also for the items that passed the test.
https://github.com/dequelabs/axe-core/blob/master/doc/API.md#section-4-performance

In vue-axe, we only use violations results among all results. The `resultType` should be set to `["violations"]` in order to avoid unnecessary calculations.

ktquez commented:
Hello @tsmd, how are you?

Thank you for your contribution!

Can you make this dynamic, which can be passed at the time of installation, through the options?

tsmd commented:
@ktquez 
Thank you!

I've changed the options to be dynamic.
I've also updated the documentation.
The `resultTypes` option is set to `['violations']` by default. This is because any types other than violations are ineffective in the current vue-axe.

Can you check again, please?

ktquez commented:
@tsmd Done!

Now available in [v1.1.0](https://github.com/vue-a11y/vue-axe/tree/v1.1.0)
Thanks.
